.github/workflows/lint.yml

```
@@ -0,0 +1,64 @@
+# CI Workflow: Lint all project components
+# Note: Cloak is a submodule — not linted here
+name: Lint
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)